sound/hda/intel-dsp-config.c

```diff
@@ -294,6 +294,10 @@ static const struct config_entry config_table[] = {
 		.flags = FLAG_SOF,
 		.device = 0xa0c8,
 	},
+	{
+		.flags = FLAG_SOF,
+		.device = 0x43c8,
+	},
```

Comment:
same comment as above, this will unconditionally select SOF. This is not what we want.